src/module/helpers/tokenUtils.mjs

```diff
@@ -0,0 +1,43 @@
+/**
+ * @param {DrawSteelActor} actor
+ * @param input
+ * @param fill
+ * @returns {Promise<void>}
+ */
+export async function showFloatyText(actor, input, fill) {
```

Comment:
I don't feel great about this being a utility function (and then it would be in the wrong folder anyway). The `Actor#_onUpdate` method is where we would want this, I think.

Reply:
And consider a name like `#displayStaminaChange` or such.